doc/design-balance-lore/VARIANTS.md

```diff
@@ -0,0 +1,20 @@
+# Variants
+
+## What are variants?
+
+Variants, added in [#51613](https://github.com/CleverRaven/Cataclysm-DDA/pull/51613) and [#46814](https://github.com/CleverRaven/Cataclysm-DDA/pull/46814) are CDDAs solution to allowing the same item have different names. It's used in guns to have generic or specific names changeable with a setting, or in items such as clothing to have lots of different colors without having a bunch of essentially duplicate entries for an item. They can be weighted, meaning different variants have a different chance of showing up when the base item is spawned. They're a great way to add in extra variety to the game at a low cost to both effort, and file sizes. Do note that for some items, such as newspapers where the description changes and not the name, [snippets](../JSON/JSON_INFO.md#snippets) may be better. There's also snippets in the form of conditional names, which can allow what are essentially variants in a more nuanced way, such as changing the name of an item depending on crafting materials, instead of a bunch of crafting recipes for different variants.  
+
+## What aren't variants?
+
+Variants cannot change any stat of an item, besides the name, description, color, sprite, and symbol. They're nothing more than an aesthetic change and have no impact on gameplay. 
+
+## Issues
+
+As cool as variants are, sometimes issues can arise if there are too many variants of an item with nothing to reign them in. As an example, condoms had a few joke flavors that didn't make sense, and this snowballed into other items having variants that were nothing more than jokes, such as chewing gum having fart, cat hairball, and zombie flavors. This doc aims to provide contributors with a good idea of what is, and isn't acceptable as variants. Another factor is that more variants complicate translating, making an already difficult job more more difficult.
```

Comment:
```suggestion
As cool as variants are, sometimes issues can arise if there are too many variants of an item with nothing to reign them in. As an example, condoms had a few joke flavors that didn't make sense, and this snowballed into other items having variants that were nothing more than jokes, such as chewing gum having fart, cat hairball, and zombie flavors. This doc aims to provide contributors with a good idea of what is, and isn't acceptable as variants. Another factor is that more variants complicate translating, making an already difficult job even more difficult.
```
You gotta proofread my proofreading or we're gonna be at this for the next 7 years